tox.ini

```diff
@@ -23,7 +23,8 @@ max-complexity = 15
 
 ignore =
     E203,  # whitespace before ':': Black disagrees with and explicitly violates this.
-    W503  # Ignore line break before binary operator, since Black violates this.
+    W503,  # Ignore line break before binary operator, since Black violates this.
+    F523,  # Ignore "%Y-%m-%d-%H-%M-%S".format(time.gmtime(time.time()))
```

Comment:
F523 is "`.format(...)` unused positional arguments" (https://flake8.pycqa.org/en/latest/user/error-codes.html)

I think you can change

```python
"%Y-%m-%d-%H-%M-%S".format(time.gmtime(time.time()))
```

to 

```python
time.strftime("%Y-%m-%d-%H-%M-%S")
```

In general, when you see linting errors, try to start by actually addressing the errors rather than getting the error message to go away.